packages/next/telemetry/storage.ts

```diff
@@ -44,13 +44,15 @@ export class Telemetry {
   private conf: Conf<any>
   private sessionId: string
   private rawProjectId: string
+  private isSrcDir?: boolean
 
-  constructor({ distDir }: { distDir: string }) {
+  constructor({ distDir, isSrcDir }: { distDir: string; isSrcDir?: boolean }) {
```

Comment:
This field should always be required, right?

Reply:
Do we want to require it for `export` and when using the `telemetry` cli command?

Reply:
Yes IMO, otherwise it defeats the purpose of having it as `meta`.

Reply:
Adding it to `next export` might be a breaking change since it would require the `pages` directory be present, we would also need to have the `dir` be passed when using the `telemetry` command to find the pages directory correctly

It seems better to not require it for these two cases or to send it a part of a different event